servicebus/data-plane/azure-servicebus/src/main/java/com/microsoft/azure/servicebus/management/QueueRuntimeInfoSerializer.java

```diff
@@ -104,6 +102,8 @@ private static QueueRuntimeInfo parseFromEntry(Node xEntry) {
                                                     case "TransferDeadLetterMessageCount":
                                                         qd.getMessageCountDetails().setTransferDeadLetterMessageCount(Long.parseLong(element.getFirstChild().getNodeValue()));
                                                         break;
+                                                    default:
```

Comment:
[SpotBugs-P2]
This method contains a switch statement where default case is missing. Usually you need to provide a default case.